zproject/dev_settings.py

```diff
@@ -62,6 +62,20 @@
 )
 
 EXTERNAL_URI_SCHEME = "http://"
+
+truthy_strings = (
+    "y",
+    "yes",
+    "t",
+    "true",
+    "on",
+    "1",
+)
```

Comment:
I borrowed this from [distutils' `strtobool`](https://github.com/pypa/distutils/blob/main/distutils/util.py#L340).

Feedback is very welcome on how to organize this bit of code.
I couldn't easily find a file that would be a good spot to place this, and didn't know if I should introduce a new file in the codebase for this change.